<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_app_Dialog</class>
 <widget class="QDialog" name="Add_app_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Appointment</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Doctor:</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Patient:</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Visit Type:</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Follow Up</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Time:</string>
      </property>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Date:</string>
      </property>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
